### ISSUE TITLE
Add optional 'Action required' Yes/No field

### ISSUE THREAD

**Issue by vnair0** [oasis-tcs/csaf#757]
As per the new CNA rules https://www.cve.org/Resources/Roles/Cnas/CNA_Rules_v4.0.pdf (effective August 8, 2024),
_4.2.3 CNAs MUST NOT consider the type of technology (e.g., cloud, on-premises, artificial intelligence, machine learning) as the sole basis for determining assignment._

This means that there will be CVEs assigned to vulnerabilities in cloud and other new technologies in which there is no action expected by a vendor from the end user. To enable end users to differentiate between CSAF advisories containing CVE's in which an action is expected (like upgrading to the latest version manually) from the 'informational' CVE's in which there is no action expected, I propose the TC considers adding an optional field 'Action Required', which can take the value Yes/No.

**Comment by santosomar** [comment]
A good example: https://msrc.microsoft.com/blog/2024/06/toward-greater-transparency-unveiling-cloud-service-cves/

**Comment by tschmidtb51** [comment]
Maybe negating it would be better: `no_action_required`

**Comment by santosomar** [comment]
I agree with the `no_action_required` suggestion. This allows us to capture the majority of the cases of CSAF disclosures where action may be required by the consumer of the document/technology.

**Comment by sthagen** [comment]
@tschmidtb51 should we add that to our final 2.1 draft version?
